addons/web/static/src/views/fields/properties/properties_field.js

```diff
@@ -296,6 +302,32 @@ export class PropertiesField extends Component {
         this.props.update(propertiesDefinitions);
     }
 
+    /**
+     * Verify that we can write on properties,
+     * if we don't have access for parent
+     */
+    async canModifyPropertiesDefinition() {
+        const definitionRecordId = this.props.record.data[this.definitionRecordField][0];
+        const definitionRecordModel = this.props.record.fields[this.definitionRecordField].relation;
+        let checkAccessRule = true;
+        try {
+            await this.orm.call(
+                definitionRecordModel,
+                "check_access_rule",
+                [definitionRecordId],
+                {
+                    operation: "write",
+                }
+            );
+        } catch (_e) {
+            this.notification.add(_lt("You need to be able to edit parent first to configure property fields"), {
+                type: "warning",
+            });
+            checkAccessRule = false;
+        }
```

Comment:
shouldn't we also change `this.state.canChangeDefinition` ? (so the pencil / create button become invisible)

Reply:
I think we don't need to change `this.state.canChangeDefinition` because AWA doesn't want to make pencil/create buttons invisible for the users that don't have access to the parent. That is why we do all this and add a notification while the user clicks on the pencil/create button. If I am wrong then please correct me @awa-odoo

Reply:
The state will be only for the current user no ? Or you mean for all parents ? (if yes I understand we don't want to do it, a bit annoying as we will have to do something on `onWillUpdateProps`)

Reply:
I think it's strange to have the buttons present and then having them disappear when used once.

We can probably assume that people are not going to click more than once after they see the error message.